### PR TITLE
Feature/add min max to non empty array

### PR DESCRIPTION
**Pull request by raveclassic**
add min and max for passed Ord instance (similar to Foldable's minimum/maximum but without unnesessary resulting Option)

please check package.json and package-lock.json (added cross-env and changed formatting to default NodeJS)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,16 +2,16 @@
...
   "main": "lib/index.js",
   "typings": "lib/index.d.ts",
   "scripts": {
     "lint": "tslint -p tsconfig.json src/**/*.ts test/**/*.ts examples/**/*.ts exercises/**/*.ts",
-    "mocha": "TS_NODE_CACHE=false mocha -r ts-node/register test/*.ts",
-    "prettier":
-      "prettier --no-semi --single-quote --print-width 120 --parser typescript --list-different \"{src,test,examples,exercises}/**/*.ts\"",
-    "fix-prettier":
-      "prettier --no-semi --single-quote --print-width 120 --parser typescript --write \"{src,test,examples,exercises}/**/*.ts\"",
+    "mocha": "cross-env TS_NODE_CACHE=false mocha -r ts-node/register test/*.ts",
```
Comment (gcanti) [inline package.json]: nice

File: src/NonEmptyArray.ts [hunk 2]
```diff
@@ -64,6 +65,12 @@ export class NonEmptyArray<A> {
   toString(): string {
     return `new NonEmptyArray(${toString(this.head)}, ${toString(this.tail)})`
   }
+  min(ord: Ord<A>): A {
```
Comment (gcanti) [inline src/NonEmptyArray.ts]: could you please add the since tag `/** @since 1.3.0 */`?

File: src/NonEmptyArray.ts [hunk 3]
```diff
@@ -64,6 +65,12 @@ export class NonEmptyArray<A> {
...
+    return fold(getMeetSemigroup(ord))(this.head)(this.tail)
+  }
+  max(ord: Ord<A>): A {
```
Comment (gcanti) [inline src/NonEmptyArray.ts]: idem

**Comment by raveclassic** [comment]
@gcanti Added some jsdoc

**Pull request merged**